linguatec_lexicon/importers.py

```diff
@@ -0,0 +1,457 @@
+import pandas as pd
+import sys
+
+from linguatec_lexicon.models import (Entry, GramaticalCategory, Word,
+                                      Lexicon, DiatopicVariation, Example,
+                                      VerbalConjugation, ImportsInfo)
+
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import CommandError
+
+from django.db import IntegrityError
+
+from linguatec_lexicon.validators import validate_column_verb_conjugation
+
+
+def get_ImportsInfo_objects(imports_info_id):
+    if imports_info_id is None:
+        return None
+    else:
+        return ImportsInfo.objects.get(pk=imports_info_id)
+
+
+def save_ImportInfo_status(imports_info, status, objects_inserted, errors):
+    if imports_info is None:
+        pass
+    else:
+        imports_info.status = status
+        if objects_inserted is not None:
+            imports_info.num_rows = objects_inserted
+        if errors is not None:
+            imports_info.errors = errors
+        imports_info.save()
+
```

Comment:
We should have 1 blank line between functions and 2 between classes (review all this file, some times there are two lines, sometimes one)

Reply:
Flake8 is not tracking those lines, i don't know if this could be because those methods are not class methods